### ISSUE TITLE
Annotation @KVBinding with GET type breaks frontend on 3.19.1

### ISSUE THREAD

**Issue by Archinamon** [rjaros/kvision#223]
Jrpc call over http from frontend to backend not delivers jrpc payload, that breaks serialization and issues a backend crash with kotlinx.serialization exception. That happens when KVBinding annotation declares GET request, then ktor do not serialize a payload body at all.

Removing annotations or changing all declared GET-routes to POST type deals a fix.

**Comment by rjaros** [comment]
Hi and thanks for your feedback!

These are in fact two different bugs in this issue. First, we completely lost GET support for Ktor during refactoring before 3.18.0. And additionally something has changed in Ktor - you will need to move `kvisionInit()` call to the end of your `main()` method to have GET requests working again:
```
fun Application.main() {
    install(Compression)
    routing {
        applyRoutes(...)
    }
    kvisionInit()
}
```
This will be fixed in the next release.

**Comment by Archinamon** [comment]
Just tested reverting KVBinding annotations to my KVService class and move kvisionInit below to the ktor routing — doesn't help.

Getting: `JsonDecodingException: Unexpected JSON token at offset 0: Expected '{, kind: CLASS'
JSON input: ` (which means empty input json).

Hope this helps.

**Comment by rjaros** [comment]
You need to wait for the next release.

**Comment by rjaros** [comment]
Released with 4.0.0
